lib/buildFactory.js

```diff
@@ -268,10 +269,15 @@ class BuildFactory extends BaseFactory {
 
                     const build = await super.create(modelConfig);
 
-                    await steps.reduce(
-                        (p, step) => p.then(() => stepFactory.create({ buildId: build.id, ...step })),
-                        Promise.resolve()
-                    );
+                    // eslint-disable-next-line no-restricted-syntax
+                    for (const step of steps) {
+                        try {
+                            // eslint-disable-next-line no-await-in-loop
+                            await stepFactory.create({ buildId: build.id, ...step });
+                        } catch (err) {
+                            logger.error(`Error in BuildFactory create -buildId:${build.id}-step:${step}`, err);
```

Comment:
can you add a space after first `-` ?
```Error in BuildFactory create - buildId```

Reply:
OK. I add space in front of error message.